R/0_installed_packages.R

```diff
@@ -0,0 +1,4 @@
+#' @title ip
+#' @description  List all user's packages on load for future reference in functions (since this is an expensive operation)
+#' @importFrom utils installed.packages
+ip <- installed.packages()
```

Comment:
move this into it's own internal env. that way you can access items more easily using a general function

```r
sinew_env <- new.env()

assign('ip',installed.packages(),envir = sinew_env)

get_env <- function(el){
  get(el,envir = sinew_env)
}
```

Reply:
better yet. this is much less expensive and gives the same chr vector

`.packages(all.available = TRUE)`

Reply:
Ooh, interesting. This is definitely the way to go. I'll add this.
The error indicates a hard coded installation path cuases the error in CI.
Is it related to this modification in the code?
Just so I understand why it failed and how to avoid it?